src/D2L.CodeStyle.Analyzers/Immutability/MutabilityInspector.cs

```diff
@@ -77,22 +77,23 @@ KnownImmutableTypes knownImmutableTypes
 			return InspectType(
 				type,
 				flags,
-				typesInCurrentCycle
+				typesInCurrentCycle,
+				type
 			);
 		}
 
 		public MutabilityInspectionResult InspectField( IFieldSymbol field ) {
 			var typesInCurrentCycle = new HashSet<ITypeSymbol>();
 
-			return InspectField( field, typesInCurrentCycle );
+			return InspectField( field, typesInCurrentCycle, default );

```

Comment:
what's ```default```?  As in, without looking at the method signature, I have no idea what you're passing ```null```/```false```/0 for

Reply:
Actually, instead of passing ```default``` everywhere, maybe just an an overload that doesn't take it.  Better then a default value, is it makes it easier to find cases where we pass it vs we don't pass it